dbt_coverage/__init__.py

```diff
@@ -616,7 +636,7 @@ def fail_compare(coverage_report: CoverageReport, compare_path: Path):
 
 def do_compute(project_dir: Path = Path('.'), cov_report: Path = Path('coverage.json'),
                cov_type: CoverageType = CoverageType.DOC, cov_fail_under: float = None,
-               cov_fail_compare: Path = None):
+               cov_fail_compare: Path = None, cov_format: Path = Path('string')):
```

Comment:
@vvvito would you mind using the [`Enum`](https://typer.tiangolo.com/tutorial/parameter-types/enum/) type in here instead? I believe it semantically makes a bit more sense than `Path` and would allow us to do a bit nice comparison on line 649 and at the same time make `cov_format` future proof (in the sense that we could add other formats in the future too 🙂)

Thanks!

Reply:
You got it boss @mrshu.
Updated all references with new enum `CoverageFormat`  on line 29. 
Two entries containing either `STRING_TABLE` or `MARKDOWN_TABLE`. Unsure of the naming convention - can be refactored in the future?

Reply:
Thanks @vvvito, that totally works 🙂